resources/views/community/components/claims/new-claims.blade.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+use LegacyApp\Community\Enums\ClaimFilters;
+
+$claimData = getFilteredClaims(
+    claimFilter: ClaimFilters::AllActiveClaims,
```

Comment:
Recommend excluding collaboration claims while you're tinkering in here.
```
    claimFilter: ClaimFilters::AllActiveClaims & ~ClaimFilters::CollaborationClaim,
```
We don't need to see multiple entries for the same game.

Reply:
Not sure if it's worth defining an `AllActivePrimaryClaims` constant for it (similar to `AllCompletedPrimaryClaims`)

Reply:
> Recommend excluding collaboration claims while you're tinkering in here.

Good idea. I've made this adjustment in https://github.com/RetroAchievements/RAWeb/commit/c2825edbcf8c4be73c349a67619991b666c00390.

> Not sure if it's worth defining an AllActivePrimaryClaims constant for it (similar to AllCompletedPrimaryClaims)

I decided against this for the time being as this would be the only consumer of such a constant; I'm not sure if this would be a premature abstraction.